src/main/java/com/spikes2212/command/genericsubsystem/commands/MoveGenericSubsystemWithPID.java

```diff
@@ -19,46 +19,29 @@
  */
 public class MoveGenericSubsystemWithPID extends Command {
 
-    /**
-     * the subsystem the command moves.
-     */
     protected final GenericSubsystem subsystem;
-
-    /**
-     * A supplier that returns the subsystem's current location.
-     */
     protected final Supplier<Double> source;
-
-    /**
-     * The PID Settings for the PID control loop.
-     */
     protected final PIDSettings pidSettings;
-
-    /**
-     * The Feed Forward Settings for the feed forward control loop.
-     */
     protected final FeedForwardSettings feedForwardSettings;
-
-    /**
-     * the setpoint for the subsystem.
-     */
     protected final Supplier<Double> setpoint;
-
-    /**
-     * An object that makes the necessary calculations for the PID control loop.
-     */
     protected final PIDController pidController;
```

Comment:
you removed the docs for this, but not like other consts, there is no docs about it in the constructor (and there shouldn't be), return the docs you deleted here